plugins/javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocIndexTest.kt

```diff
@@ -2,14 +2,64 @@ package org.jetbrains.dokka.javadoc
 
 import org.jetbrains.dokka.javadoc.pages.IndexPage
 import org.jetbrains.dokka.javadoc.renderer.TemplateMap
+import org.jetbrains.dokka.links.DRI
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
+import kotlin.test.assertNotNull
 
 internal class JavadocIndexTest : AbstractJavadocTemplateMapTest() {
 
+    private val genericQuery = """
```

Comment:
I thought about `generic in the programming context

Reply:
Will try to think of a better name